site/assets/scss/_clipboard-js.scss

```diff
@@ -35,3 +38,7 @@
     background-color: $primary;
   }
 }
+
+.btn-edit {
+  right: 3.65rem;
```

Comment:
That makes me think we may benefit from a container around the two buttons to better handle position and alignment.

Reply:
Agreed

Reply:
I did that. Check out my latest changes